src/app/layout/footer/footer.component.ts

```diff
@@ -24,9 +23,6 @@ export class FooterComponent implements OnInit {
     _platformInfo.get().subscribe((platformInfo) => {
       this.platform = platformInfo
     })
```

Comment:
It seems like some or all of the components related to this PR do not require the TogglezService to inject anymore on the constructor.